exercises/change/change_test.exs

```diff
@@ -0,0 +1,44 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("change.exs")
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule ChangeTest do
+  use ExUnit.Case
+
+  test "returns :error on empty list" do
+    assert Change.generate(1, []) == :error
+  end
+
+  @tag :pending
+  test "generates the correct change when only one coin type is needed" do
+    change = %{1 => 5, 10 => 0}
+    assert Change.generate(5, [1, 10]) == {:ok, change}
+  end
+
+  @tag :pending
+  test "generates the correct change when multiple coin types are needed" do
+    change = %{1 => 3, 5 => 1, 10 => 1}
+    assert Change.generate(18, [1, 5, 10]) == {:ok, change}
+  end
+
+  @tag :pending
+  test "returns error when it is not possible to generate change" do
```

Comment:
Should be "returns :error when it is not possible to generate change" to be consistent with the first test.